### DISCUSSION TITLE
Flashing scrollbar

### DISCUSSION THREAD

**Discussion by K4LCIFER** [xournalpp/xournalpp#2642] [Q&A] [answered]
I write in Xournal++ using a Wacom tablet, and what I find is that each time I touch my pen to the surface of the tablet to write in Xournal++ the scroll bar disappears. What this means is that each time I write a stroke the scrollbar disappears and reappears. This creates a really annoying flashing scrollbar effect when writing. Since one is of course writing many strokes and each stroke causes the scrollbar to flash, this can end up being very distracting. 
What I'm looking for is some  way to disable the disappearing scrollbar when writing. I just want it to statically stay there, and not disappear.

**Comment by K4LCIFER** [comment, marked as answer]
I have found the solution: `Edit->Preferences->View->Scrollbars->Disable-Scrollbar-Fade-Out`

**Reply by iammihirsig** [reply]
You really save me bro .... lot's of thanks 🙏
